frontend/src/components/dashboard/subdomain/SearchForm.tsx

```diff
@@ -33,6 +34,11 @@ export const SubdomainSearchForm = (props: Props) => {
     props.onPick(subdomainInput);
   };
 
+  const onInput: FormEventHandler<HTMLInputElement> = async (event: any) => {
```

Comment:
As mentioned on Slack, this `FormEventHandler<HTMLInputElement>` should be a `ChangeEventHandler<HTMLInputElement>`, after which the `any` can be removed.